src/dex/hashflow/hashflow.ts

```diff
@@ -82,13 +91,66 @@ export class Hashflow extends SimpleExchange implements IDex<HashflowData> {
       token !== undefined,
       'Hashflow auth token is not specified with env variable',
     );
+
     this.hashFlowAuthToken = token;
-    this.api = new HashflowApi('taker', 'paraswap', this.hashFlowAuthToken);
+    this.api = new HashflowApi(
+      'taker',
+      HASHFLOW_API_CLIENT_NAME,
+      this.hashFlowAuthToken,
+    );
+
+    this.pricesCacheKey = `${this.dexHelper.config.data.network}_${this.dexKey}_prices`;
+    this.marketMakersCacheKey = `${this.dexHelper.config.data.network}_${this.dexKey}_mms`;
```

Comment:
I think better to prefix with CACHE_PREFIX as well. It is global for dex-lib to know which service is serving current cache value

Reply:
@Verisana added